sunpy/lightcurve/sources/sit.py

```diff
@@ -0,0 +1,325 @@
+    >>> sit = lc.SITLightCurve.create(TimeRange('2012/06/01', '2012/06/05'), "ahead", "1min", "4He")
+    
+    References
+    ----------
+    | http://www.srl.caltech.edu/STEREO/Public/SIT_public.html
+
+    """
+    
+    # def peek(self, title="GOES Xray Flux"):
+    #     """Plots GOES light curve is the usual manner"""
+    #     figure = plt.figure()
+    #     axes = plt.gca()
+
+    #     dates = matplotlib.dates.date2num(parse_time(self.data.index))
+
+    #     axes.plot_date(dates, self.data['xrsa'], '-',
+    #                  label='0.5--4.0 $\AA$', color='blue', lw=2)
+    #     axes.plot_date(dates, self.data['xrsb'], '-',
+    #                  label='1.0--8.0 $\AA$', color='red', lw=2)
+
+    #     axes.set_yscale("log")
+    #     axes.set_ylim(1e-9, 1e-2)
+    #     axes.set_title(title)
+    #     axes.set_ylabel('Watts m$^{-2}$')
+    #     axes.set_xlabel(datetime.datetime.isoformat(self.data.index[0])[0:10])
+
+    #     ax2 = axes.twinx()
+    #     ax2.set_yscale("log")
+    #     ax2.set_ylim(1e-9, 1e-2)
+    #     ax2.set_yticks((1e-9, 1e-8, 1e-7, 1e-6, 1e-5, 1e-4, 1e-3, 1e-2))
+    #     ax2.set_yticklabels((' ', 'A', 'B', 'C', 'M', 'X', ' '))
+
+    #     axes.yaxis.grid(True, 'major')
+    #     axes.xaxis.grid(False, 'major')
+    #     axes.legend()
+
+    #     # @todo: display better tick labels for date range (e.g. 06/01 - 06/05)
+    #     formatter = matplotlib.dates.DateFormatter('%H:%M')
+    #     axes.xaxis.set_major_formatter(formatter)
+
+    #     axes.fmt_xdata = matplotlib.dates.DateFormatter('%H:%M')
+    #     figure.autofmt_xdate()
+    #     figure.show()
+
+    #     return figure
+
+    @classmethod
+    def _get_default_url(cls):
+        """
+           Retrieve latest SIT data for 1 minute average of 4He specie from ahead orientation 
+           ( if no other data is specified )
+
+        """
+        now = datetime.datetime.utcnow()
+        time_range = TimeRange(datetime.datetime(now.year, now.month, now.day), now)
+        url_does_exist = net.url_exists(cls._get_url_for_date_range(time_range))
+        while not url_does_exist:
+            time_range = TimeRange(time_range.start-datetime.timedelta(days=1),
+                                   time_range.start)
+            url_does_exist = net.url_exists(cls._get_url_for_date_range(time_range))
+        return cls._get_url_for_date_range(time_range, "ahead", '1min', "4He")
+
+    
+
+    @staticmethod
+    def _get_url_for_date_range(*args):
+        """Returns a list of URLs to the SIT data for the specified range of dates along with the 
+         tuple containing specified start and end dates .
+
+        Parameters
+        ----------
+        args : TimeRange, datetimes, date strings
+            Date range should be specified using a TimeRange, or start
+            and end dates at datetime instances or date strings.
+
+        instrument_orientation : string
+            SIT instrument orientation (default = "ahead")
```

Comment:
I would not call this `orientation` as it relates to the spacecraft where it is.  [STEREO](http://www2.physik.uni-kiel.de/stereo/downloads/STEREO_orbit.png) is a mission with two spacecraft, one is ahead of earth and the other behind, therefore the name.  I would call this `stereo_spacecraft` or something like that.